staging/src/k8s.io/apiserver/pkg/apis/audit/types.go

```diff
@@ -77,8 +77,6 @@ type Event struct {
 	// AuditLevel at which event was generated
 	Level Level
 
-	// Time the request reached the apiserver.
-	Timestamp metav1.Time
```

Comment:
This explains what CreationTimestamp is:
```
// CreationTimestamp is a timestamp representing the server time when this object was
// created. It is not guaranteed to be set in happens-before order across separate operations.
// Clients may not set this value. It is represented in RFC3339 form and is in UTC.
```
Since having two timestamp doesn't do harm, I think we can obey the definition of CreationTimestamp, and not mutate CreationTimestamp after the object is created.(I know users can treat event in different stages as different objects, but it is one object for us developers)
What about this:
1.  in the request received stage:
     CreationTimestamp = StageTimestamp = time.Now()
2. in later stages:
    StageTimestamp is mutated to new time.


So that StageTimestamp.Sub(CreationTimestamp) shows how much time it has cost when request comes to this stage.